libr/main/radare2.c

```diff
@@ -392,6 +392,10 @@ R_API int r_main_radare2(int argc, const char **argv) {
 		r_sys_crash_handler ("gdb --pid %d");
 		free (tmp);
 	}
+	if ((tmp = r_sys_getenv ("R_LOG_INIT_LVL"))) {
+		r_log_set_level (atoi (tmp));
```

Comment:
are negative values also valid? maybe worth adding a check and an error message if dimmed